Configurations/15-android.conf

```diff
@@ -29,18 +29,18 @@
                 $ndk = $ENV{$ndk_var};
                 last if defined $ndk;
             }
-            die "\$ANDROID_NDK_HOME is not defined"  if (!$ndk);
-            if (!-d "$ndk/platforms" && !-f "$ndk/AndroidVersion.txt") {
-                # $ndk/platforms is traditional "all-inclusive" NDK, while
-                # $ndk/AndroidVersion.txt is so-called standalone toolchain
-                # tailored for specific target down to API level.
-                die "\$ANDROID_NDK_HOME=$ndk is invalid";
+            die "\$ANDROID_NDK_ROOT is not defined" if (!$ndk);
```

Comment:
On line 27 the list of envvars checked is unchanged: `qw(ANDROID_NDK_HOME ANDROID_NDK)`, why is the error message changing from complaining about `ANDROID_NDK_HOME` to `ANDROID_NDK_ROOT`?

Reply:
That's the variable name that is used on master and I didn't notice it was different. Thanks for pointing that out. I don't know why they're different, but I guess I'll put the old name back.

Reply:
Done